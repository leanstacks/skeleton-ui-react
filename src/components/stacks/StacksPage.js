import React from 'react';

import Stack from './Stack';

const StacksPage = (props) => (
  <div className="page">
    <div className="container">
      <div className="row mt-5 mb-3">
        <div className="col-12">
          <h2 className="display-4">Technology Stacks</h2>
        </div>
      </div>
      <div className="row mb-3">
        <div className="col-12 col-md-4 mb-3 mb-md-0">
          <Stack name="Application" type="application" />
        </div>
        <div className="col-12 col-md-4 mb-3 mb-md-0">
          <Stack name="Test" type="test" />
        </div>
        <div className="col-12 col-md-4 mb-3 mb-md-0">
          <Stack name="Build" type="build" />
        </div>
      </div>
    </div>
  </div>
);


export default StacksPage;