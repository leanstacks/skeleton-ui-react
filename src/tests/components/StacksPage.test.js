import React from 'react';
import { shallow } from 'enzyme';

import StacksPage from '../../components/stacks/StacksPage';

test('should render StacksPage correctly', () => {
  const wrapper = shallow(<StacksPage />);
  expect(wrapper).toMatchSnapshot();
});