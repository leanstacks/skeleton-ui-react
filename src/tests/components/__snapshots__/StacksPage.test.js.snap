// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render StacksPage correctly 1`] = `
<div
  className="page"
>
  <div
    className="container"
  >
    <div
      className="row mt-5 mb-3"
    >
      <div
        className="col-12"
      >
        <h2
          className="display-4"
        >
          Technology Stacks
        </h2>
      </div>
    </div>
    <div
      className="row mb-3"
    >
      <div
        className="col-12 col-md-4 mb-3 mb-md-0"
      >
        <Connect(Stack)
          name="Application"
          type="application"
        />
      </div>
      <div
        className="col-12 col-md-4 mb-3 mb-md-0"
      >
        <Connect(Stack)
          name="Test"
          type="test"
        />
      </div>
      <div
        className="col-12 col-md-4 mb-3 mb-md-0"
      >
        <Connect(Stack)
          name="Build"
          type="build"
        />
      </div>
    </div>
  </div>
</div>
`;
